README.md

```diff
@@ -29,29 +29,9 @@ Our goal is to allow event organizers to set up a professional conference websit
 | **Blog** | post announcements, updates and useful information |
 
 ## Getting Started
-1. [Fork repository](https://github.com/gdg-x/hoverboard/fork) and clone your fork locally
-1. Install [Node.js (v10+)](https://nodejs.org/en/download/)
-1. Install project dependencies: `npm install`
-1. Create [Firebase account](https://console.firebase.google.com) and login into [Firebase CLI](https://firebase.google.com/docs/cli/): `npx firebase login`
-1. Update [Hoverboard config](/config) and [Resources](/data)
-1. Import initial data to the Firebase Database
-    * Generate `serviceAccount.json` file
-      - Go to [console.firebase.google.com](https://console.firebase.google.com) -> Project settings -> Service accounts
-      - Ensure that **Node.js** is selected and press **Generate new private key**
-      - Read the warning and press **Generate key**
-      - Save the file as `serviceAccount.json` and to the root of your hoverboard directory (❗Do NOT commit this file to the public repository)
-    * Enable Firestore in web console at [console.firebase.google.com](https://console.firebase.google.com) -> Database -> Cloud Firestore -> Create database. Select **locked mode** and press **Enable**
-    * [Optional] You can edit `docs/default-firebase-data.json)` file using your own data
-	  * Select your Firebase project `npx firebase use <YOUR_PROJECT_ID>`
-    * Run `npm run firestore:init`
-1. Run locally
-   * `npm start`
-1. Build and deploy
-   * `npm run build`
-   * `npm run deploy`
-
-*NOTE:* By default command using configurations from `/configs/development.json`.
-To serve locally or deploy the production app use `npm run start:prod` and `npm run deploy:prod` respectively.
+
+:first_quarter_moon_with_face: Read the [Set Up Guide](/docs/tutorials/00-set-up.md).
+
 
 :book: Read the [Full Setup Guide](/docs/).
```

Comment:
The names "Set Up Guide" and "Full Setup Guide" are confusing. Maybe rename this to "Documentation"?